components/style/properties/longhand/inherited_text.mako.rs

```diff
@@ -334,7 +334,7 @@ ${helpers.single_keyword("text-align-last",
         if input.try(|input| input.expect_ident_matching("normal")).is_ok() {
             Ok(SpecifiedValue::Normal)
         } else {
-            specified::Length::parse_non_negative(input).map(SpecifiedValue::Specified)
+            specified::Length::parse_any(input).map(SpecifiedValue::Specified)
```

Comment:
Instead of creating this function for this specific case, let's use `Length::parse(context, input)`.